### PR TITLE
Fix/login feature not protecting routes

### PR DESCRIPTION
**Pull request by juskek**

### CODE REVIEW

File: front/next.config.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
 const nextConfig = {
     experimental: {
         typedRoutes: true,
-        serverComponentsExternalPackages: ['@prisma/client','bcrypt']
+        // serverComponentsExternalPackages lists packages that should not be called by the client (see https://youtu.be/2kgqPvs0j_I?t=1285)
```
Comment (Lerri-Cofannos) [inline front/next.config.js:5]: Is it intentional to have left this comment here?

File: front/README.md
Comment (Lerri-Cofannos) [inline front/README.md:1]: Updating the documentation is a nice reflex!

**Review by Lerri-Cofannos — APPROVED** [review]
Nice work! The comment I left is not blocking

File: front/next.config.js (hunk 1 above)
Comment (juskek) [reply front/next.config.js:5]: yes, it's not explicit in the docs that adding packages to that field ensures that those packages are only used server side

**Pull request merged**